netcore/gen-xunit-runner/README.md

```diff
@@ -4,15 +4,37 @@ This is a small netcore app to convert a corefx test assembly into a console app
 
 # Usage
 
+## Project Runner Generator
+
 ```
-dotnet run ../runner.cs ~/git/corefx/ ~/git/corefx//artifacts/bin/System.Runtime.Tests/netcoreapp-Unix-Debug/System.Runtime.Tests.dll -notrait category=nonosxtests -notrait category=failing -notrait category=Outerloop -stoponfail
+dotnet run ../output ~/git/corefx/ ~/git/corefx/artifacts/bin/System.Runtime.Tests/netcoreapp-Unix-Debug/System.Runtime.Tests.dll -notrait category=nonosxtests -notrait category=failing -notrait category=Outerloop -stoponfail
 ```
 
 There is support for response files, i.e.
 ```
-dotnet run ../runner.cs ~/git/corefx/ ~/git/corefx//artifacts/bin/System.Runtime.Tests/netcoreapp-Unix-Debug/System.Runtime.Tests.dll @excludes.rsp
+dotnet run ../output ~/git/corefx/ ~/git/corefx/artifacts/bin/System.Runtime.Tests/netcoreapp-Unix-Debug/System.Runtime.Tests.dll @excludes.rsp
+```
+
+
+## Tests Runner Build
+
+Go to for example `../output` from the example above and do
+
+```
+dotnet build
 ```
 
+## Tests Execution
+
+This assumes you have sucesfully ran HelloWorld sample before
+
+```
+../../sample/dotnet  --fx-version "3.0.0-preview-27408-5" bin/Debug/netcoreapp3.0/{Name of the test-dll} .dll
+```
+
+The test dll will usually be in the format `System.Runtime.Tests-runner.dll`
+
+
 # Notes
 
 - If xunit can't laod a trait discoverer assembly, it silently ignores the error.
```

Comment:
laod -> load? (on one more place in .cs) ... was bothering me a bit, but didn't seem worth a separate PR.